packages/core/src/main/Account.ts

```diff
@@ -400,6 +400,7 @@ export class Account<T = any> extends EventEmitter {
         this.mlsConfig,
         entropyData,
       );
+      this.coreCryptoClient.registerCallbacks({authorize: () => true, clientIdBelongsToOneOf: () => true});
```

Comment:
TODO: implement correct checks (if the user is admin of the conversation).
The info needs to come from the consumer side, so we will need to add callbacks to the constructor of `Account`